api_server_temperature.py

```diff
@@ -0,0 +1,54 @@
+import csv
+from datetime import datetime 
+import boto3
+import json 
+
+'''
+Function for getting the path prefix for accesing the mean temperature file in the S3
+'''
+
+def get_object_path_prefix():
+    today = datetime.now()
+    month_day = today.strftime("%m-%d")
+    date = str(today.year-2) + "-" + month_day
+    path_prefix = 'mean_temperatures/' + date 
+    return path_prefix
+    
+'''
+Function for getting mean temperature file name, which is automatically generated by hadoop in the spark job
+'''
```

Comment:
Docstring formatting 
https://www.mybluelinux.com/python-docstrings-annotations-and-pep8/